crates/composer_api/src/lib.rs

```diff
@@ -15,8 +15,16 @@ pub const DEFAULT_SERVER_ADDRESS: &str = "localhost:8888";
 #[derive(Serialize, Deserialize, Debug)]
 pub enum Event {
     TestTick,
-    StdoutWrite { length: usize },
-    StderrWrite { length: usize },
+    TimedTestTick {
+        /// timestamp as Duration since the UNIX_EPOCH
+        ts: Duration,
```

Comment:
```suggestion
        timestamp: Duration,
```
Better to be explicit, especially in APIs.